src/llamacrawl/services/readers/elasticsearch.py

```
@@ -113,18 +112,23 @@ def __init__(
                 "field_mappings": self.field_mappings,
                 "batch_size": self.batch_size,
-                "timestamp_field": self.timestamp_field
-            }
+                "timestamp_field": self.timestamp_field,
+            },
         )
 
-    def load_data(self, **kwargs: Any) -> list[Document]:
+    def load_data(
+        self,
+        progress_callback: Callable[[int, int], None] | None = None,
```

Ruff (0.14.0)

Unused method argument: `progress_callback`

(ARG002)